crates/holochain/src/core/workflow/initialize_zomes_workflow.rs

```diff
@@ -45,7 +45,7 @@ pub async fn initialize_zomes_workflow<Ribosome>(
     args: InitializeZomesWorkflowArgs<Ribosome>,
 ) -> WorkflowResult<InitResult>
 where
-    Ribosome: RibosomeT + Send + 'static,
+    Ribosome: RibosomeT + Clone + Send + Sync + 'static,
```

Comment:
If we require `Clone` also, consider promoting it per the previous comment

Reply:
@neonphog this one doesn't seem to be supported by mockall crate